apps/extension/src/offscreen.ts

```diff
@@ -71,7 +71,11 @@ const spawnWorker = ({
       worker.removeEventListener('message', onWorkerMessage);
       worker.terminate();
       console.error('Worker ErrorEvent', ev);
-      reject(ev.error ?? new Error(`Worker ErrorEvent ${filename}:${lineno}:${colno} ${message}`));
+      reject(
+        ev.error instanceof Error
+          ? ev.error
+          : new Error(`Worker ErrorEvent ${filename}:${lineno}:${colno} ${message}`),
+      );
```

Comment:
Reviewers: [the v6.19.0 release](https://github.com/typescript-eslint/typescript-eslint/releases/tag/v6.19.0) of the TypeScript ESLint plugin introduced a [rule](https://eslint.org/docs/latest/rules/prefer-promise-reject-errors) that Promise rejections should contain an `Error` instance. I assume we're preferring to stick to the default ESLint config in general, so I updated this code to adhere to the rule. However, I have no idea how to test it manually :)